contrib-viz/03.contrib-stats.ipynb

```diff
@@ -0,0 +1,481 @@
+    "stats['merged_pull_requests'] = result['total_count']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "https://api.github.com/search/issues?q=repo%3Agreenelab%2Fcovid19-review+type%3Apr+is%3Aunmerged+state%3Aclosed&sort=created&order=asc\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Closed PRs that were not merged\n",
+    "result = github_issue_search('repo:greenelab/covid19-review type:pr is:unmerged state:closed')\n",
+    "stats['declined_pull_requests'] = result['total_count']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "https://api.github.com/search/issues?q=repo%3Agreenelab%2Fcovid19-review+type%3Apr+state%3Aopen&sort=created&order=asc\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Open PRs\n",
+    "result = github_issue_search('repo:greenelab/covid19-review type:pr state:open')\n",
+    "stats['open_pull_requests'] = result['total_count']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "https://api.github.com/search/issues?q=repo%3Agreenelab%2Fcovid19-review+type%3Aissue+state%3Aopen&sort=created&order=asc\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Open Issues\n",
+    "result = github_issue_search('repo:greenelab/covid19-review type:issue state:open')\n",
+    "stats['open_issues'] = result['total_count']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "https://api.github.com/search/issues?q=repo%3Agreenelab%2Fcovid19-review+type%3Aissue+state%3Aclosed&sort=created&order=asc\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Closed Issues\n",
+    "result = github_issue_search('repo:greenelab/covid19-review type:issue state:closed')\n",
+    "stats['closed_issues'] = result['total_count']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Write stats"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\n",
+      "  \"creation_time_utc\": \"2021-08-27T21:21:39.116795\",\n",
+      "  \"creation_date_pretty\": \"August 27, 2021\",\n",
+      "  \"commit\": \"2c88132aa6e259244b474470bfe72722f89f1683\",\n",
+      "  \"total_commits\": 4876,\n",
+      "  \"manuscript_commits\": 2775,\n",
+      "  \"review_authors\": 52,\n",
+      "  \"github_stars\": 112,\n",
+      "  \"github_forks\": 75,\n",
+      "  \"merged_pull_requests\": 559,\n",
```

Comment:
Do we want to include `merged_pull_requests` and the number of issues in the methods manuscript?  That's nice to show how active the project has been.

Reply:
That's great idea! Added a sentence to the conclusion in #1033